### PR TITLE
Add created_at and updated_at for switches

### PR DESCRIPTION
**Pull request by vQuadX**

### CODE REVIEW

File: its_on/models.py [hunk 1]
```diff
@@ -17,6 +18,13 @@
     sa.Column('groups', postgresql.ARRAY(sa.String(255))),
     sa.Column('version', sa.Integer, nullable=True),
     sa.Column('comment', sa.Text),
+    sa.Column('created_at', sa.DateTime, default=lambda: datetime.datetime.now(), nullable=True),
```
Comment (vladimir-kotikov) [inline its_on/models.py:21]: You'd better use `server_default` and `functions.now()` to make sure datetimes are consistent and not depend on application server's time settings. See [here](https://stackoverflow.com/a/33532154/4113568) for more details

File: its_on/models.py [hunk 2]
```diff
@@ -17,6 +18,13 @@
...
+    sa.Column(
+        'updated_at', sa.DateTime,
+        default=lambda: datetime.datetime.now(),
+        onupdate=lambda: datetime.datetime.now(),
```
Comment (vladimir-kotikov) [inline its_on/models.py:25]: Same here about using DB server datetime instead of application server's local one

File: tests/test_api_switch.py [hunk 3]
```diff
@@ -46,9 +46,9 @@ async def test_switch_filter_by_version(version, expected_result, setup_tables_a
 
 
 async def test_switches_full_info(
-    setup_tables_and_data, client, switches_full_info_expected_result,
+    setup_tables_and_data, client, switches_full_info_expected_result, mocker,
```
Comment (vladimir-kotikov) [inline tests/test_api_switch.py:49]: Redundant `mocker` argument?

File: its_on/models.py (hunk 1 above)
Comment (sergey-jerzy) [reply its_on/models.py:21]: `default=datetime.datetime.utcnow`

Comment (vQuadX) [reply its_on/models.py:21]: > `default=datetime.datetime.utcnow`

lambda for datetime mocking


> You'd better use `server_default` and `functions.now()` to make sure datetimes are consistent and not depend on application server's time settings. See [here](https://stackoverflow.com/a/33532154/4113568) for more details

1. server_onupdate don't set "ON UPDATE CURRENT_TIMESTAMP" - https://docs.sqlalchemy.org/en/13/core/defaults.html#marking-implicitly-generated-values-timestamps-and-triggered-columns (it working only for mysql)
2. Using datetime i control default values with code and can mock it

Comment (vladimir-kotikov) [reply its_on/models.py:21]: > server_onupdate don't set "ON UPDATE CURRENT_TIMESTAMP"

I think you've misread the docs - they say the opposite: this basically doesn't work for MySQL out of the box (although there's a workaround for it) but works for everything else. As for the second argument - you technically can use freezegun to control time during tests so mocking it is not really needed.

That said I think this is generally better to use db time but if you think that server time works here as well - i'm fine with that?

**Review by vladimir-kotikov — APPROVED** [review]

**Pull request merged**